packages/earth-admin/src/components/inline-edit-card/InlineEditCard.tsx

```diff
@@ -58,12 +58,16 @@ export default function InlineEditCard(props: InlineCardProps) {
     validForm,
   } = props;
 
-
   const [isEditing, setIsEditing] = useState(false);
   const [isLoading, setIsLoading] = useState(false);
   const [serverErrors, setServerErrors] = useState(null);
 
   const state = isEditing ? 'open' : 'close';
+
+  useEffect(() => {
+    setServerErrors(null);
+  }, [isEditing])
+
```

Comment:
Instead of relying on useEffect, you could make it more explicit by resetting the errors on cancel click